static/app/components/overlayArrow.tsx

```diff
@@ -1,79 +1,105 @@
+import {forwardRef, ForwardRefRenderFunction, HTMLAttributes} from 'react';
 import styled from '@emotion/styled';
+import {Placement} from '@popperjs/core';
```

Comment:
I was trying to avoid importing `@popperjs/core` by just using he `PopperProps` type. Mostly for ease of refactoring later.